src/modules/visitor-space/components/MaterialRequestBlade/MaterialRequestBlade.tsx

```diff
@@ -101,6 +114,7 @@ const MaterialRequestBlade: FC<MaterialRequestBladeProps> = ({
 				),
 			});
 			onSuccesCreated();
+			refetchMaterialRequests();
```

Comment:
do we still need this, since the error is thrown before we even open this blade?